docs/sdk/1-workshop/06-claim.md

```diff
@@ -2,14 +2,15 @@
 id: claim
 title: 💬 Claim
 ---
+
 import CodeBlock from '@theme/CodeBlock';
-import Example1 from '!!raw-loader!../../../code-examples/3_1_claim.ts';
-import Example2 from '!!raw-loader!../../../code-examples/3_2_claim.ts';
+import Example1 from '!!raw-loader!../../../code-examples/4_1_claim.ts';
+import Example2 from '!!raw-loader!../../../code-examples/4_2_claim.ts';
 
 In this section, you'll play the role of a <span class="label-role claimer">claimer</span>.
 
 - You'll first make a claim about yourself in the form of a `Claim` object;
-- But a claim in itself has no value. To become valid in the eyes of <span class="label-role verifier">verifiers</span>, it needs to be attested by an entity that <span class="label-role verifier">verifiers</span> trust: an <span class="label-role attester">attester</span>. So you'll create a `RequestForAttestation` object from your `Claim` object, so that an <span class="label-role attester">attester</span> can attest it.
+- But a claim in itself has no value. To become valid in the eyes of <span class="label-role verifier">verifiers</span>, it needs to be attested by an entity that <span class="label-role verifier">verifiers</span> trust: an <span class="label-role attester">attester</span>. So you'll create a `RequestForAttestation` object from your `Claim` object, sign it with the DID, so that an <span class="label-role attester">attester</span> can attest it.
```

Comment:
```suggestion
- But a claim in itself has no value.
To become valid in the eyes of <span class="label-role verifier">verifiers</span>, it needs to be attested by an entity that <span class="label-role verifier">verifiers</span> trust: an <span class="label-role attester">attester</span>.
So you'll create a `RequestForAttestation` object from your `Claim` object, sign it with the DID, so that an <span class="label-role attester">attester</span> can attest it.
```